Sources/AGGRenderer/CPPAGGRenderer/CPPAGGRenderer.cpp

```diff
@@ -347,6 +347,7 @@ namespace CPPAGGRenderer{
       agg::conv_transform<agg::path_storage, agg::trans_affine> trans(rect_path, matrix);
       agg::conv_curve<agg::conv_transform<agg::path_storage, agg::trans_affine>> curve(trans);
       agg::conv_stroke<agg::conv_curve<agg::conv_transform<agg::path_storage, agg::trans_affine>>> stroke(curve);
+      stroke.width(thickness);
```

Comment:
What I believe is happening here is that all of the previous test cases that have been tested don't take into account the stroke width when using the draw_line function, so when I add this line, virtually all of the AGGRenderer test cases take the L.